easybuild/easyconfigs/q/QuickFF/QuickFF-2.0.1-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "PythonPackage"
+
+name = 'QuickFF'
+version = '2.0.1'
+
+homepage = 'http://molmod.github.io/QuickFF/'
+description = """QuickFF is a Python package developed at the Center for
+Molecular Modeling (CMM) to quickly derive accurate force fields from ab initio
+calculations."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/molmod/QuickFF/releases/download/v%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+python = "Python"
+pythonversion = '2.7.10'
+pythonshortversion = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('matplotlib', '1.5.0', versionsuffix),
+    ('molmod', '1.1', versionsuffix),
+    ('yaff', 'thesissven2.13', '%s-HDF5-1.8.15-patch1-serial' %(versionsuffix))
```

Comment:
Indeed. the dependency needs to fixed. This worked on our side because of the many different versions that were installed.